contrib/docker/docker-compose.yml

```diff
@@ -1,4 +1,8 @@
 # docker-compose build && docker-compose up
+# If you experience problems with the CKAN container not being
+# able to connect to the DB, then most likely the DB has not
+# started up quickly enough. Just do "docker-compose up ckan"
+# again to retry
```

Comment:
probably this can be done from separate container which will configure ckan using https://github.com/jwilder/dockerize since we also need to be sure that all redis, solr, pg are up and running. I'll try to provide PR